types/react-aria-menubutton/index.d.ts

```diff
@@ -0,0 +1,151 @@
+// Type definitions for react-aria-menubutton 5.0
+// Project: https://github.com/davidtheclark/react-aria-menubutton
+// Definitions by: Muhammad Fawwaz Orabi <https://github.com/forabi>
+//                 Chris Rohlfs <https://github.com/crohlfs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import * as React from "react";
+
+export interface WrapperState {
+	isOpen: boolean;
+}
+
+export type WrapperProps<T extends HTMLElement> = React.HTMLAttributes<T> & {
```

Comment:
I know it's mainly a stylistic thing, but could you change these to interfaces that extend `React.HTMLAttributes<T>`? It tends to be more efficient and ensures that properties declared in extended interfaces are verified against each other.

Reply:
Good call, changed.